### DISCUSSION TITLE
长期求助文献传递

### DISCUSSION THREAD

**Discussion by kanition** [kanition/pbrtbook#41] [问答]
因为笔者~将~(已经)毕业，~可能~今后再无渠道访问学术资源，所以极有可能无法阅读到文中的参考文献以改进翻译。在此特向各位求助无偿提供访问渠道，或者通过邮箱传递单篇指定文献。本人在此诚挚感谢大家的无私帮助！

传递文献（建议为pdf格式）请发送：jiangjun_1995@qq.com

**Comment by indevn** [comment]
我尚是在校学生。如果日后对某些学术资源有需求，可以联系我(indevn@outlook.com)代为访问。

**Reply by kanition** [reply]
感谢您的无私帮助！需要时我会联系您的。谢谢Thanks♪(･ω･)ﾉ

**Comment by HakkerBarry** [comment]
我目前也是学生，身在国外，笔者有任何需求也可以联系我。zixuanzhang1998@gmail.com或者是学校邮箱 zixuan.zhang@utah.edu。

**Reply by kanition** [reply]
谢谢您的无私帮助♪(･ω･)ﾉ，必要时我会联系您的

**Reply by akatokikwok** [reply]
http://libgen.rs/
老毛子的共产主义PDF网站,我RTR4就搁上面搜的,还带sha1

**Reply by kanition** [reply]
非常感谢分享了这么一个宝藏网站！我一搜还真有，说不定以后再自学一下实时渲染吧

**Comment by Connaught0** [comment]
你好，我也是在校学生预期在校3年，有事可以联系yaosx2020@gmail.com

**Reply by kanition** [reply]
谢谢您的无私帮助，以后若有需求我会与您联系~

**Comment by kanition** [comment]
https://raw.githubusercontent.com/bookyue/my_books/master/mathematics/calculus/stewart-calculus/Calculus%20Early%20Transcendentals%209th%20Edition%20by%20James%20Stewart%2C%20Daniel%20K.%20Clegg%2C%20Saleem%20Watson%2C%20Lothar%20Redlin.pdf.pdf

**Comment by kanition** [comment]
工程光学
https://ktxr.rs/zlib2/pilimi-zlib2-17250000-17339999/17316220.pdf
https://zh.annas-archive.org/md5/c63cbdae2aff66021e968c074a64fd6f

**Comment by kanition** [comment]
https://libgen.li/

**Comment by kanition** [comment]
Introduction to Probability Models
https://libgen.li/ads48c2e54659903e85b5d8aea1ecbc6ba998GUNLJY

**Comment by kanition** [comment]
https://fuxiii.github.io/Essentials.of.Vulkan/StartFromVulkanSDK.html

**Comment by kanition** [comment]
Understanding Diffusion Models: A Unified Perspective
https://arxiv.org/pdf/2208.11970

Denoising Diffusion Probabilistic Models
https://hojonathanho.github.io/diffusion/
https://hojonathanho.github.io/diffusion/assets/denoising_diffusion20.pdf
https://github.com/lucidrains/denoising-diffusion-pytorch

Tutorial on Diffusion Models for Imaging and Vision
https://arxiv.org/pdf/2403.18103

**Comment by kanition** [comment]
https://github.com/diff-usion/Awesome-Diffusion-Models

**Reply by kanition** [reply]
https://nn.labml.ai/diffusion/stable_diffusion/sampler/ddim.html

**Comment by kanition** [comment]
The Ins and Outs of Solving Quadratic Equations with Floating-Point Arithmetic
https://cnrs.hal.science/hal-04116310v1/document

**Reply by kanition** [reply]
https://perso.ens-lyon.fr/jean-michel.muller/slides-split.pdf

**Reply by kanition** [reply]
https://www.cl.cam.ac.uk/teaching/1011/FPComp/fpcomp10slides.pdf

**Comment by kanition** [comment]
平方根
http://www.lomont.org/papers/2003/InvSqrt.pdf

**Comment by kanition** [comment]
cmake
https://cmake.org/cmake/help/book/mastering-cmake/index.html

**Comment by kanition** [comment]
TOTP
https://github.com/pyauth/pyotp

**Reply by kanition** [reply]
https://github.com/beemdevelopment/Aegis

**Reply by kanition** [reply]
https://github.com/nextcloud/server

**Comment by kanition** [comment]
search web
https://annas-archive.org/

**Comment by kanition** [comment]
Electromagnetic Fields And Interactions
https://libgen.li/ads.php?md5=811d2418c88aab46feea3a31065104a3
